lib/astarte_core/triggers/policy_protobuf/handler.pb.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Astarte.Core.Triggers.PolicyProtobuf.Handler.StrategyType do
+  @moduledoc false
+
+  use Protobuf, enum: true, protoc_gen_elixir_version: "0.11.0", syntax: :proto3
+
+  def fully_qualified_name, do: "Handler.StrategyType"
+
+  field :INVALID, 0
+  field :DISCARD, 1
+  field :RETRY, 2
+end
+
+defmodule Astarte.Core.Triggers.PolicyProtobuf.Handler do
+  @moduledoc false
+
+  use Protobuf, protoc_gen_elixir_version: "0.11.0", syntax: :proto3
+
+  def fully_qualified_name, do: "Handler"
+
+  oneof(:on, 0)
```

Comment:
Import the protobuf library in `.formatter.exs` so `oneof` doesn't get parenthesized